spec/services/user_signup_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'spec_helper'
+
+describe UserSignup do
+  describe "#sign_up", :vcr do
+    context "valid personal info and valid card" do
+       before do
+         charge = double(:charge, successful?: true)
+         StripeWrapper::Charge.should_receive(:create).and_return(charge)
+       end
+
+      it "should create the user" do
```

Comment:
I would recommend to not use `should` word in the example message - read more [here](http://www.betterspecs.org/#should).